universal-application-tool-0.0.1/test/services/applicant/ApplicantDataTest.java

```diff
@@ -38,6 +38,18 @@ public void preferredLocale_defaultsToEnglish() {
     assertThat(data.preferredLocale()).isEqualTo(Locale.US);
   }
 
+  @Test
```

Comment:
Question (non-blocking): is this from a merge conflict or an intended change? no need to change, just curious since this seems unrelated.

Reply:
Ah I forgot about this - there was a point where I started to doubt that data was being overwritten, so I  added a test. I'll keep it in since it isn't hurting anything, but you're right it is somewhat unrelated